packages/client/ui/Dialog/DialogContent.tsx

```diff
@@ -13,7 +13,7 @@ export const DialogContent = React.forwardRef<
     <RadixDialog.Content
       ref={ref}
       className={twMerge(
-        'fixed top-[50%] left-[50%] max-h-[85vh] w-[90vw] max-w-md translate-x-[-50%] translate-y-[-50%] rounded-lg bg-white p-4 shadow-dialog focus:outline-none data-[state=open]:animate-contentShow',
+        'md:w-xl fixed top-[50%] left-[50%] max-h-[85vh] w-[95vw] max-w-[95vw] translate-x-[-50%] translate-y-[-50%] rounded-lg bg-white p-4 shadow-dialog focus:outline-none md:max-w-xl',
```

Comment:
+1 I think there needs to be more padding to make the modal look the same as the old one. Here are the two next to each other:

<img width="665" alt="Screenshot 2023-07-31 at 12 57 10" src="https://github.com/ParabolInc/parabol/assets/39854876/0b6af975-18d8-46a5-b968-34dbfa070280">
<img width="660" alt="Screenshot 2023-07-31 at 12 57 39" src="https://github.com/ParabolInc/parabol/assets/39854876/39732e02-2361-4da3-bc7e-e5909b43b8cf">